client/components/foldable-card/index.jsx

```diff
@@ -96,6 +96,10 @@ var FoldableCard = React.createClass( {
 	},
 
 	renderContent: function() {
+		if ( ! this.state.expanded ) {
```

Comment:
I like that way too, but it was pointed out to me on past PRs that it's not the preferred way on Calypso. Instead, the recommendation was to do `{ this.state.expanded ? <Component /> : null }`.

If it's ok I'll be more than happy to change that to `{ ... && ... }`... and start using it again :wink: